sdk/src/main/java/com/horizen/account/api/rpc/response/RpcResponseSuccess.java

```diff
@@ -14,6 +14,11 @@ public RpcResponseSuccess(RpcId id, Object result) {
         this.result = result;
     }
 
+    @JsonInclude
```

Comment:
do we need `@JsonProperty("result")` above in line 9?

Reply:
no, the _JsonProperty_ is not needed, it is now removed from the `RpcResponseSuccess` class. It is the same in the `RpcResponseError` class with the _error_ parameter.